### DISCUSSION TITLE
Can we use SAM2 with text prompt?

### DISCUSSION THREAD

**Discussion by ibrahimnzamane** [opengeos/segment-geospatial#322] [Q&A] [answered]
I have been experimenting segment-geospatial and have tried SAM2, SAM, SAM_HQ, and SAMLang. I want to ask is it possible to use LangSAM with SAM2 as it is being used with SAM?

**Comment by giswqs** [comment, marked as answer]
Not yet. It will be added soon

**Reply by ibrahimnzamane** [reply]
Okay, Thank you.
